### PR TITLE
[15.0][ADD] stock_valuation_layer_accounting_date

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This PR is a continuation of https://github.com/OCA/stock-logistics-warehouse/pull/1593 which was closed.

@qrtl

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_valuation_layer_accounting_date/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* `Quartile Limited <info@quartile.co>`_:
+  * Aung Ko Ko lin
```
Comment (yostashiro) [inline stock_valuation_layer_accounting_date/readme/CONTRIBUTORS.rst:3]:
```suggestion

  * Aung Ko Ko lin
```

**Comment by AungKoKoLin1997** [comment]
pre-commit is failed and I created another PR to fix this. https://github.com/OCA/stock-logistics-warehouse/pull/1881

**Review by yostashiro — APPROVED** [review]
The module has been in production use for a few months.

**Comment by AungKoKoLin1997** [comment]
@rousseldenis Can we get this PR merged?

**Pull request merged**